PocketKit/Sources/PocketKit/MyList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -529,17 +531,14 @@ extension SavedItemsListViewModel: SavedItemsControllerDelegate {
         for type: NSFetchedResultsChangeType,
         newIndexPath: IndexPath?
     ) {
-        guard .update == type else {
-            return
-        }
-
```

Comment:
Do we still want to keep this?

Reply:
yup, youre right. oversight on me for not adding that back in